plugins/collection/class.tx_dlf_collection.php

```diff
@@ -95,53 +95,33 @@ public function main($content, $conf) {
      * @return	string		The list of collections ready to output
      */
     protected function showCollectionList() {
-
```

Comment:
Please preserve those empty lines throughout the file (there are many occasions where you removed them).